src/window/editor/scenario_event_details.c

```diff
@@ -185,7 +185,8 @@ static unsigned int count_maximum_needed_list_items(void)
 {
     unsigned int total_items = 0;
     scenario_condition_group_t *group;
-    array_foreach(data.event->condition_groups, group) {
+    array_foreach(data.event->condition_groups, group)
+    {
```

Comment:
```suggestion
    array_foreach(data.event->condition_groups, group) {
```

I know by default visual studio separates the lines, but try to keep these at the same line if possible 😉